src/main/java/trivia/GameBetterWithLogging.java

```diff
@@ -3,11 +3,7 @@
 // REFACTOR ME
 public class GameBetterWithLogging implements IGame {
 
-  private Questions questions = new Questions(
-      QuestionCategoryName.POP.createQuestionList(),
-      QuestionCategoryName.SCIENCE.createQuestionList(),
-      QuestionCategoryName.SPORTS.createQuestionList(),
-      QuestionCategoryName.ROCK.createQuestionList());
+  private Questions questions = Questions.create();
```

Comment:
.create should receive how much questions Game has, and it can be placed as constant